README.md

```diff
@@ -134,6 +126,14 @@ assigned from the Source.
 | `table`     | the name of a table in the database that the connector should write to, by default | **true** | `users`                                     |
 | `keyColumn` | column name uses to detect if the target table already contains the record         | **true** | `id`                                        |
 
+## Type convention
+
+Type convention describes the conversion between Oracle to Go types.
```

Comment:
Maybe we can add information than Oracle db doesn't have boolean type that's why we do this conversion

Reply:
I've added the column with the explanations.